src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Workloads/WorkloadDescriptor.cs

```diff
@@ -28,12 +34,12 @@ public WorkloadDescriptor(string workloadName, string visualStudioComponentId)
         /// <summary>
         ///     Gets the Visual Studio setup component ID corresponding to the .NET workload.
         /// </summary>
-        public string VisualStudioComponentId { get; }
+        public ISet<string> VisualStudioComponentIds { get; }
```

Comment:
If this doesn't have to be mutable, use `IReadOnlySet<string>`.

Reply:
That is a CPS-internal interface